src/CollectionIdentifier_TEST.cc

```diff
@@ -164,3 +164,10 @@ TEST(CollectionIdentifier, AsPrettyString)
     EXPECT_NE(str.find("raspberry"), std::string::npos);
   }
 }
+
+//////////////////////////////////////////////////
+int main(int argc, char **argv)
```

Comment:
Just curious, is it necessary to have the main here?

Reply:
Nope, it was just one of the things I tried. It follows the same pattern as `ModelIdentifier` and `WorldIdentifier`